.ci/integration_test.sh

```diff
@@ -58,14 +59,23 @@ func main() {
 		fmt.Printf("Dial Client failed, err:%v", err)
 		return
 	}
-	address, tx, instance, err := DeployMain(client.GetTransactOpts(), client)
+	address, tx, instance, err := Deploy${struct}(client.GetTransactOpts(), client)
 	if err != nil {
 		fmt.Printf("Deploy failed, err:%v", err)
 		return
 	}
 	fmt.Println("contract address: ", address.Hex()) // the address should be saved
 	fmt.Println("transaction hash: ", tx.Hash().Hex())
-	hello := &MainSession{Contract: instance, CallOpts: *client.GetCallOpts(), TransactOpts: *client.GetTransactOpts()}
+EOF
+}
+
+generate_hello() {
+    local struct="${1}"
+    local output="${2}"
+    generate_main "${1}" "${2}"
+cat << EOF >> ${output}
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Double quote to prevent globbing and word splitting.](https://app.codacy.com/gh/FISCO-BCOS/go-sdk/pullRequest?prid=4970769)